gc/gc-iceberg-mock/build.gradle.kts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  `java-library`
+  `maven-publish`
+  signing
+  `nessie-conventions`
+}
+
+extra["maven.name"] = "Nessie - GC - Mocked Iceberg data for tests"
```

Comment:
I am not sure whether this module was really required. Because it is very easy to have end to end Iceberg local file system testcases and validate the files against it. 

I feel it takes an extra effort to maintain the Iceberg spec code (as we need to keep track of the changes)